src/converter.ts

```diff
@@ -131,9 +131,54 @@ export default async function (ast: ParseResult<File>, targetedDiscordModApiLibr
          */
         debugger;
         // console.log(findAllTypesWithPath(element, "MemberExpression"));
-        const paths = findPathsToType({ obj: element, targetType: "MemberExpression" });
-        for (let index2 = 0; index2 < paths.length; index2++) {
-            const element2 = paths[index2];
+        const newExpressionPaths = findPathsToType({ obj: element, targetType: "NewExpression" });
+        for (let index2 = 0; index2 < newExpressionPaths.length; index2++) {
+            const element2 = newExpressionPaths[index2];
+            const trueObj = deepFind<NewExpression>(element, element2);
+            console.log(trueObj);
```

Comment:
Remove debug

Reply:
Solved in 75f11cde88db9e69536aec80def47096b37eefa6